app/src/main/res/values/colors.xml

```diff
@@ -40,12 +40,13 @@
   <color name="black_alpha_10">#1a000000</color>
   <color name="black_alpha_17">#2B000000</color>
   <color name="black_alpha_30">#4D000000</color>
-  <color name="black_alpha_50">#88000000</color>
+  <color name="black_alpha_50">#80000000</color>
   <color name="blue_darken_10">@color/ksr_cobalt_600</color>
   <color name="gray_alpha_50">@color/ksr_grey_300</color>
   <color name="gray_darken_10">@color/ksr_grey_500</color>
-  <color name="green_alpha_20">#33037362</color>
-  <color name="green_alpha_50">#80037362</color>
+  <color name="green_alpha_18">#2e009E74</color>
```

Comment:
My bad; I've realised that `green_alpha_18` is not accessible with our primary green as text on top of it. Could we please make and use `green_alpha_10` instead?